kvm-snapshot-explanation (#113)
apache/cloudstack-documentation: source/adminguide/storage.rst

#. Click the Snapshot button. |SnapshotButton.png|

KVM volume Snapshot specifics
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

In recent CloudStack versions, by default, creating a volume snapshot for a running VM is disabled
due to a possible volume corruption in certain cases. To enable creating a volume snapshots while the VM

> is running, the global setting 'kvm.snapshot.enabled' must be set to 'True'.

Reviewer: ```suggestion
is running, the global setting 'kvm.snapshot.enabled' must be set to 'True'.

```

Author: I don't see any differences here?

> The way the volume snapshot is now created has been changed. 

Reviewer: ```suggestion
The volume snapshot creation has changed in recent versions:

```
